frontend/src/index.test.js

```diff
@@ -2505,27 +2505,25 @@ test('test makeGraph() - diagram case', () => {
 });
 
 // TODO:
-test('test makeGraph() - 1 cset "single small"', () => {
-  console.log();
-});
-
-// TODO:
-test('test makeGraph() - 2+ csets', () => {
-  console.log();
-});
+// test('test makeGraph() - 1 cset "single small"', () => {
+// });
+//
+// // TODO:
+// test('test makeGraph() - 2+ csets', () => {
+// });
 
 // - roots, leaves
 // todo?
 // test('test roots, leaves - diagram case', () => {});
 
 test('test roots, leaves - 1 cset "single small"', () => {
   const gc = new GraphContainer(singleSmallGraphContainerGraphData);
-  expect(gc.roots).toStrictEqual(["442793", "4044391", "4129519", "isolated"]);
+  expect(gc.roots).toStrictEqual(["442793", "4044391", "4129519", "unlinked"]);
```

Comment:
## Bug fix: `isolated` --> `unlinked`
I needed to update this to what you had in `GraphState.jsx`. I wrote this on Feb 7, and you wrote this unlinked bit on Feb 4, so the code was probably rapidly changing, and our branches were out of sync.

```js
    this.graph.addNode('unlinked');
    this.nodes['unlinked'] = unlinkedConceptsParent;
    for (let c of this.unlinkedConcepts) {
      this.graph.addDirectedEdge('unlinked', c);
    }
```

@Sigfried However, I don't really understand what it's trying to say here. What does it mean to have a root called `unlinked`? I can understand a concept being unlinked, but it should still have a concept ID. Note that graph objects we're passing to `GraphContainer` don't have anything inside them called 'unlinked' or 'isolated'.

Reply:
`unlinked` becomes the root of all orphans. they get inserted below that